docker-compose.yaml

```diff
@@ -24,3 +24,8 @@ services:
       - .env
     ports:
       - ${POSTGRES_PORT}:5432
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U youruser"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
```

Comment:
I added this health check because the etl in the `app` service was trying to connected to the database before it was ready.